addons/l10n_ec/models/l10n_ec_sri_payment.py

```diff
@@ -8,6 +8,9 @@ class SriPayment(models.Model):
 
     _name = "l10n_ec.sri.payment"
     _description = "SRI Payment Method"
+    _order = 'active DESC, sequence, id'
```

Comment:
Actually I would not order on active at all.  We have filters to show the ones with or without.  So please just remove this line.